### PR TITLE
Add content to radiation closets and medicine lockers

### PR DESCRIPTION
**Pull request by Lamrr**
## About the PR 
The currently implemented versions of filled radiation closets and medicine lockers are empty. I've changed this, adding rad suits to the radiation closet and some medical supplies to the medicine lockers, as well as giving catchy descriptions to both. This should hopefully result in less engineers dying from the singularity and provide a source of medicine to doctors when the NanoMeds aren't working.

**Changelog**

:cl:
- tweak: Radiation closets and medicine lockers now come with supplies inside.

### CODE REVIEW

File: Resources/Prototypes/Catalog/Fills/Lockers/medical.yml [hunk 1]
```diff
@@ -2,6 +2,20 @@
   id: LockerMedicineFilled
   suffix: Filled
   parent: LockerMedicine
+  description: "Ze healing leaves little time for ze hurting."
+  components:
+  - type: StorageFill
+    contents:
+      - id: BoxPillCanister
```
Comment (Elijahrane) [inline Resources/Prototypes/Catalog/Fills/Lockers/medical.yml:9]: Why pill canisters? Doctors have nothing to fill them with and chemists have them in the chemistry closet.

**Review by Elijahrane — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/Fills/Lockers/medical.yml (hunk 1 above)
Comment (Lamrr) [reply Resources/Prototypes/Catalog/Fills/Lockers/medical.yml:9]: Medicine lockers are bit of buffet of medical junk, so I think it's reasonable to give the doctors some pill canisters so they can build their own pill collection.

**Review by ShadowCommander — APPROVED** [review]

File: Resources/Prototypes/Catalog/Fills/Lockers/engineer.yml [hunk 2]
```diff
@@ -97,4 +97,14 @@
...
\ No newline at end of file
+  suffix: Filled
+  description: "More comfortable then radiation poisioning."
+  components:
+  - type: StorageFill
+    contents:
+      - id: ClothingHeadHatHoodRad
+        amount: 2
+      - id: ClothingOuterSuitRad
+        amount: 2
```
Comment (mirrorcult) [inline Resources/Prototypes/Catalog/Fills/Lockers/engineer.yml:108]: put some mesons in this too

**Review by mirrorcult — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/Fills/Lockers/engineer.yml (hunk 2 above)
Comment (Lamrr) [reply Resources/Prototypes/Catalog/Fills/Lockers/engineer.yml:108]: Done

**Review by PaulRitter — APPROVED** [review]

File: Resources/Prototypes/Catalog/Fills/Lockers/engineer.yml [hunk 3]
```diff
@@ -97,4 +97,16 @@
 - type: entity
   id: ClosetRadiationSuitFilled
   parent: ClosetRadiationSuit
-  suffix: Filled
\ No newline at end of file
+  suffix: Filled
+  description: "More comfortable then radiation poisioning."
```
Comment (ElectroJr) [inline Resources/Prototypes/Catalog/Fills/Lockers/engineer.yml:101]: Not sure if mistake, or implying that the rad suits do nothing so you might as well get comfortable first.

Comment (Lamrr) [reply Resources/Prototypes/Catalog/Fills/Lockers/engineer.yml:101]: I imagine that radiation suits are not very comfortable to wear. None the less that's better (more comfortable) then not wearing them and getting radiation poison.

Comment (ElectroJr) [reply Resources/Prototypes/Catalog/Fills/Lockers/engineer.yml:101]: then -> than

Comment (Lamrr) [reply Resources/Prototypes/Catalog/Fills/Lockers/engineer.yml:101]: Ah gotcha, I'll push a correction to the spelling.

**Pull request merged**